.gitignore

```diff
@@ -35,3 +35,4 @@ dask*/
 /.venv
 /.devcontainer
 tmp/
+nbtest1.ipynb
```

Comment:
Why is this necessary in the gitignore file?

Reply:
indeed it's better to exclude test notebooks from git, instead of ignoring them @moezali1